.github/workflows/reusable-codeql-analysis.yml

```diff
@@ -0,0 +1,25 @@
+name: "Code Scanning with GitHub CodeQL"
+
+on:
+  workflow_call:
+
+jobs:
+  package-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+          cache: yarn
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v2
+      - name: Install dependencies
+        run: yarn
+      - name: Build graphql
+        run: yarn build
+        working-directory: packages/graphql
```

Comment:
Only for the graphql package?

Reply:
To start with yes. I first want to see how it all works before expanding it to other packages